Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM alpine:3.7
-MAINTAINER Nic Cope <n+docker@rk0n.org>
+FROM node as node
+ADD frontend/ .
+RUN npm install && npm run build
+
+FROM golang as golang
+WORKDIR /go/src/github.com/negz/kuberos/
+ENV GOOS="linux" GOARCH="amd64" CGO_ENABLED=0
+
+ADD . .
+COPY --from=node dist/ dist/frontend
+COPY --from=node index.html dist/frontend/
 
-RUN apk update && apk add ca-certificates
+RUN go get -u github.com/Masterminds/glide && \
+  go get -u github.com/rakyll/statik && \
+  glide install
 
-COPY "dist/kuberos" /
\ No newline at end of file
+RUN cd statik && go generate && cd ..
+RUN go build -o /kuberos ./cmd/kuberos
+
+FROM alpine:3.7 as ca-certificates
+RUN apk --no-cache add ca-certificates
+
+FROM scratch
+MAINTAINER Nic Cope <n+docker@rk0n.org>
+COPY --from=ca-certificates /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=golang /kuberos /
+ENTRYPOINT ["/kuberos"]
```

Comment:
I'd rather not add an `ENTRYPOINT` because I believe doing so would be a breaking change for those who already run Kuberos via a Kubernetes manifest.

Reply:
I understand, while I would like to see and `ENTRYPOINT` it's more of a nice to have. I'll set that aside.